queries.sql

```diff
@@ -66,3 +66,230 @@ SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+ 13 | Boarmon    | 2005-06-07    |               7 | t        |      20.4 | digimon
+  7 | Pikachu    | 2021-01-07    |               1 | f        |     15.04 | pokemon
+ 11 | Squirtle   | 1993-04-02    |               3 | f        |    -12.13 | pokemon
+ 14 | Blossom    | 1998-10-13    |               3 | t        |        17 | pokemon
+  9 | Charmander | 2020-02-08    |               0 | f        |       -11 | pokemon
+(10 filas)
+*/
+
+UPDATE animals SET weight_kg=weight_kg*-1 
+WHERE weight_kg<0;
+SELECT * FROM animals;
+/*
+ id |    name    | date_of_birth | escape_attempts | neutered | weight_kg | species 
+----+------------+---------------+-----------------+----------+-----------+---------
+  1 | Agumon     | 2020-02-03    |               0 | t        |     10.23 | digimon
+  6 | Gabumon    | 2018-11-15    |               2 | t        |         8 | digimon
+  8 | Devimon    | 2017-05-12    |               5 | t        |        11 | digimon
+ 13 | Boarmon    | 2005-06-07    |               7 | t        |      20.4 | digimon
+  7 | Pikachu    | 2021-01-07    |               1 | f        |     15.04 | pokemon
+ 14 | Blossom    | 1998-10-13    |               3 | t        |        17 | pokemon
+ 10 | Plantmon   | 2021-11-15    |               2 | t        |       5.7 | digimon
+ 12 | Angemon    | 2005-06-12    |               1 | t        |        45 | digimon
+ 11 | Squirtle   | 1993-04-02    |               3 | f        |     12.13 | pokemon
+  9 | Charmander | 2020-02-08    |               0 | f        |        11 | pokemon
+*/
+COMMIT;
+
+/* ****************************************************************************************************** */
+
+/* How many animals are there? */
+SELECT COUNT(*) FROM animals;
+/*
+ count 
+-------
+    10
+(1 fila)
+*/
+
+/* How many animals have never tried to escape? */
+SELECT COUNT(*) FROM animals 
+WHERE escape_attempts=0;
+/*
+ count 
+-------
+     2
+(1 fila)
+*/
+
+/* What is the average weight of animals? */
+SELECT AVG(weight_kg) FROM animals;
+/*
+---------------------
+ 15.5500000000000000
+(1 fila)*/
+
+/* Who escapes the most, neutered or not neutered animals? */
+SELECT neutered, SUM(escape_attempts) FROM animals 
+GROUP BY neutered;
+/*
+ neutered | sum 
+----------+-----
+ f        |   4
+ t        |  20
+(2 filas)
+*/
+
+/* What is the minimum and maximum weight of each type of animal? */
+SELECT species, MIN(weight_kg) FROM animals 
+GROUP BY species;
+/*
+ species | min 
+---------+-----
+ pokemon |  11
+ digimon | 5.7
+(2 filas)
+*/
+
+SELECT species, MAX(weight_kg) FROM animals 
+GROUP BY species;
+/*
+ species | max 
+---------+-----
+ pokemon |  17
+ digimon |  45
+(2 filas)
+*/
+
+/* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
+SELECT species,AVG(escape_attempts) FROM animals 
+WHERE date_of_birth 
+BETWEEN 'Jan 1, 1990' AND 'Dec 31, 2000'
+GROUP BY species;
```

Comment:
- [ ] Kindly make sure your above query also include animals born` 'Jan 1, 1990'` and `'Dec 31, 2000'` as the current one excludes both of them. To fix that just use `>=` and `<=` instead of `BETWEEN`. For more detail you can refer this **[article](https://stackoverflow.com/questions/1630239/sql-between-vs-and/1630254)**

- [ ] Kindly make sure you also add new screenshots of all data you previously wrote wrongly, i.e. screenshots of your new changes

Reply:
BETWEEN includes the date that is selected, I tried this before taking the capture. Here is an
 example:
![Captura desde 2023-02-20 21-28-44](https://user-images.githubusercontent.com/113067895/220240415-a7b04ca4-9978-44c0-9dba-6993b3b34e22.png)

Notice that the animals with the same birthday of the filter are in the list